elements/x-tooltip/x-tooltip.js

```diff
@@ -92,8 +92,16 @@ class XTooltipElement extends HTMLElement {
         clientRect || this._element.getBoundingClientRect();
       this._x = elementClientRect.left;
 
+      let pos = Settings.get(Settings.keys.BOOKMARKS_DRAWER_POSITION);
+
       const tooltipClientRect = this.$tooltip.getBoundingClientRect();
-      const x = this._x - (tooltipClientRect.width - 10);
+
+      let x = this._x - (tooltipClientRect.width - 10);
```

Comment:
I think it's time this `10` becomes a named constant `TOOLTIP_X_OFFSET`.  It was already kind of mysterious, and now we are using it in two places (the `54` uses it implicitly)